src/main/java/org/breedinginsight/services/UserService.java

```diff
@@ -291,4 +292,27 @@ public User updateRoles(AuthenticatedUser actingUser, UUID userId, SystemRolesRe
     public boolean exists(UUID id){
         return dao.existsById(id);
     }
+
+    //TODO: Remove once registration flow is complete
+    public User updateOrcid(AuthenticatedUser activeUser, UUID userId, OrcidRequest orcidRequest) throws DoesNotExistException, AlreadyExistsException {
+
+        // This is a temporary fix so any authenticated user is able to update any users orcid.
+
+        BiUserEntity biUser = dao.fetchOneById(userId);
+
+        if (biUser == null) {
+            throw new DoesNotExistException("UUID for user does not exist");
+        }
+
+        List<BiUserEntity> biUserWithOrcidList = dao.fetchByOrcid(orcidRequest.getOrcid());
+        for (BiUserEntity biUserWithOrcid: biUserWithOrcidList){
+            if (!biUserWithOrcid.getId().toString().equals(userId.toString())){
```

Comment:
I think you could do equals on UUID if you wanted.

Reply:
changed and pushed